### PR TITLE
Responsive navbar #77

### PR DESCRIPTION
**Pull request by tpokora**
Mistake in branch name

### CODE REVIEW

**Comment by tomek199** [comment]
Tests!

**Comment by tpokora** [comment]
Tests fixed

File: elo-rating-frontend/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "chart.js": "^2.7.1",
     "core-js": "^2.5.3",
     "ng2-cookies": "^1.0.12",
+    "ngx-bootstrap": "^3.1.1",
```
Comment (tomek199) [inline elo-rating-frontend/package.json:36]: Please update also **package-lock.json** file

Comment (tpokora) [reply elo-rating-frontend/package.json:36]: Done

**Pull request merged**